independent-publisher-connectors/DPIRD Radar/Readme.md

```diff
@@ -0,0 +1,53 @@
+
+## DPIRD Radar Connector for Western Australia
+The Department of Primary Industries and Regional Development's (DPIRD) allows you to integrate the RADAR estimated rainfall data within your applications.
+
+You can see what information is available from the API, and how you can filter and search for specific information, by exploring the [interactive home page](https://api.dpird.wa.gov.au/v2/radar/openapi/).
+
+## Prerequisites
+You will need the following to proceed
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+* The Power platform CLI tools
+* An [API key](https://www.agric.wa.gov.au/form/dpird-api-registration) from DPIRD in accordance with [terms and conditions](https://www.agric.wa.gov.au/apis/api-terms-and-conditions)
+
+## Authentication

```

Comment:
There is no Obtaining credentials section which is a required section in the readme file. You can always refer to this [template ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md)for more information on the readme file.